### ISSUE TITLE
DataFrame.info() issue with duplicated columns

### ISSUE THREAD

**Issue by wavexx** [pydata/pandas#11761]
When duplicated columns exists, DataFrame.info() returns broken output:

````
pd.DataFrame([[1, 1]], columns=['x','x']).info()
<class 'pandas.core.frame.DataFrame'>
Int64Index: 1 entries, 0 to 0
Data columns (total 2 columns):
x    1 non-null x    int64
x    int64
dtype: object
x    1 non-null x    int64
x    int64
dtype: object
dtypes: int64(2)
memory usage: 24.0 bytes
````

See #11754

**Comment by jreback** [comment]
hmm, that is a problem....ok

**Comment by jreback** [comment]
closed via #11772
